This patch fixes the manpage so it displays properly on Solaris.  Without this
patch, the "mak pkg-config" command shows no output.
--- pkg-config-0.23/pkg-config.1-orig	2013-07-08 01:48:06.890795558 -0500
+++ pkg-config-0.23/pkg-config.1	2013-07-08 01:50:15.263772090 -0500
@@ -2,27 +2,7 @@
 .\" pkg-config manual page.
 .\" (C) Red Hat, Inc. based on gnome-config man page (C) Miguel de Icaza (miguel@gnu.org)
 .\"
-.
-.\" Macros to disable groff line adjustment warnings that we can't easily
-.\" fix in the text.
-.ie \n(.g \{\
-.  de DW
-.    nr .oldwarn \n[.warn]
-.    warn 0
-.  .
-.  de EW
-.    warn \n[.oldwarn]
-.  .
-.\}
-.el \
-.\{ \
-.  de DW
-.  .
-.  de EW
-.  .
-.\}
-.
-.TH pkg-config 1
+.TH pkg-config 1 "7 Jul 2013"
 .SH NAME
 pkg-config \- Return metainformation about installed libraries
 .SH SYNOPSIS
